docs/install/ipfs-companion.md

```diff
@@ -7,17 +7,24 @@ description: The IPFS Companion browser extension allows you to interact with yo
 
 IPFS Companion allows you to interact with your IPFS node and the extended IPFS network through your browser. The add-on is available for Brave, Chrome, Edge, Firefox, and Opera. It enables support for `ipfs://` addresses, automatically loads websites and file paths from an IPFS gateway, allows you to easily import and share a file with IPFS, and more.
 
-IPFS Companion works in tandem with an IPFS node running on your local machine, so make sure you have a [node installed](ipfs-desktop.md) before installing this add-on.
+## Prerequsities
```

Comment:
I like that you're adding this `Prerequsities` section. I'd note that you can still use IPFS companion without a local node though you lose some of the functionality.

For example, you can load websites with DNSLink via a gateway. 

@lidel WDYT?

Reply:
@2color updated the copy based on your point ^ https://github.com/ipfs/ipfs-docs/pull/1444/files#diff-4a2f396b026959f2b7e270fdba96eb753a21a68f5895eb3f4fd20e9666a7ec4fR12-R18